tests/remote_attestation/maa/sgxlkl_enclave/tlscli.c

```diff
@@ -0,0 +1,455 @@
+#define _GNU_SOURCE
```

Comment:
I can't comment on binary files, so I'll do it here.
How were the sgxlkl_cert.der and sgxlkl_private_key.pem files generated? If it can't be easily automated, please add a README explaining why and how to create the files manually.

Reply:
This is temp solution. We will use sgx-lkl to generate those certificates later.